install/docker-compose.yaml

```diff
@@ -0,0 +1,25 @@
+version: '3.8'
+
+services:
+  cache:
+    image: memcached:1.6-alpine
+    restart: always
+  database:
+    build:
+      context: ./postgres
+      args:
+        FILES_DOMAIN: 'wjfiles.test'
+        POSTGRES_PASSWORD: 'wikijump-postgres'
+    restart: always
+  php-fpm:
+    build:
+      context: ./php-fpm
+      args:
+        MAIN_DOMAIN: 'wikijump.test'
+        FILES_DOMAIN: 'wjfiles.test'
```

Comment:
I'm good with this in the interim, long term we're going to want to replace these with environment variables.

Reply:
These are `ARG` directives, which from what I can tell are essentially the same as `ENV` to Docker. The idea here is they can be overridden within the `docker-compose.yaml` file.

But yeah, if there's a cleaner and less duplicative way of doing this, I'm all ears.